### ISSUE TITLE
Fix tests on main branch

### ISSUE THREAD

**Issue by michielbdejong** [ledgerloops/ledgerloops-earthstar#2]

**Comment by michielbdejong** [comment]
```
1 import * as Earthstar from "https://deno.land/x/earthstar@v10.2.2/mod.ts";
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/server/sync_test.ts:12:2 - error TS2304: Cannot find name 'Deno'.
```

Maybe I can remove the dependency on Deno?

**Comment by michielbdejong** [comment]
Hm, looking into Deno a bit it sounds like the main drawback for me at this point is it doesn't handle types for npm  modules well, but it also [sounds like this was fixed in v1.27](https://stackoverflow.com/questions/74005494/how-to-include-definitely-typed-library-to-my-deno-project). I'll give it a go!

**Comment by michielbdejong** [comment]
Installing the Deno extension for VSCode and running Deno:enable like in https://github.com/denoland/vscode_deno fixed the errors
